app/src/pages/view_map.py

```diff
@@ -1,21 +1,29 @@
-"""
-https://fema-ffrd.github.io/stac-browser/#/external/duwamish-pilot.s3.amazonaws.com/stac/duwamish/collection.json?.language=en
-"""
```

Comment:
I think it would help to have the SST transposition domain as a polygon on the map. (Not for this PR.)